Source/Build Specs/MeasurementLink Service.vipb

```diff
@@ -19,6 +19,7 @@
     <Package_Dependencies>
       <Additional_External_Dependencies>ni_lib_labview_grpc_library =1.0.0.1</Additional_External_Dependencies>
       <Additional_External_Dependencies>ni_lib_labview_grpc_servicer =1.0.0.1</Additional_External_Dependencies>
+      <Additional_External_Dependencies>ni_protobuf_types &gt;=1.0.0.7</Additional_External_Dependencies>
```

Comment:
Brad is changing this back to 1.0.0.1 in another PR so you should change this version here as well